go.mod

```diff
@@ -324,7 +324,7 @@ require (
 
 replace (
 	github.com/cosmos/cosmos-sdk => github.com/celestiaorg/cosmos-sdk v1.8.0-sdk-v0.46.7
-	github.com/filecoin-project/dagstore => github.com/celestiaorg/dagstore v0.0.0-20221014072825-395797efb659
+	github.com/filecoin-project/dagstore => github.com/celestiaorg/dagstore v0.0.0-20230404123415-177451f83136
```

Comment:
Nit: we need to tag dagstore impl if it is not going into upstream

Reply:
It is going into upstream, there is a PR right now to add it :)